jdi-light-mobile-tests/src/main/java/io/github/com/sections/ContactForm.java

```diff
@@ -0,0 +1,20 @@
+package io.github.com.sections;
+
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.interfaces.complex.IsCombobox;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.*;
+import io.github.com.entities.Contacts;
+
+public class ContactForm extends Form<Contacts> {
+	TextField name, lastName, position, passportNumber, passportSeria;
+
+	Dropdown gender;
+	IsCombobox religion;
+
+	Checkbox passport, acceptConditions;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)